android/java/res/layout/brave_sync_done.xml

```diff
@@ -58,44 +58,49 @@
-             android:layout_height="wrap_content"
-             android:layout_width="match_parent"
-
-             android:background="@drawable/brave_sync_warning"
-             android:padding="10dp"
-             android:layout_marginStart="15dp"
-             android:layout_marginEnd="15dp"
-             android:layout_marginBottom="15dp" >
-
-             <ImageView android:id="@+id/brave_sync_image_not_working"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:src="@drawable/ic_sync_error_48dp"
-                android:scaleType="center"
-                android:contentDescription="@null"
-                android:layout_gravity="center" />
-
-                <LinearLayout android:orientation="vertical"
-                     android:layout_height="wrap_content"
-                     android:layout_width="match_parent" >
-
-                     <TextView
-                         android:text="@string/sync_error_card_title"
-                         android:layout_width="wrap_content"
-                         android:layout_height="wrap_content"
-                         android:gravity="left"
-                         android:textSize="15sp" />
-
-                     <TextView
-                         android:text="@string/brave_sync_android_sync_disabled"
-                         android:layout_width="wrap_content"
-                         android:layout_height="wrap_content"
-                         android:gravity="left"
-                         android:textSize="12sp" />
-                </LinearLayout>
-        </LinearLayout>
+        <TableLayout android:id="@+id/brave_sync_btn_android_sync_settings"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="@drawable/brave_sync_warning"
+            android:padding="10dp"
+            android:layout_marginStart="15dp"
+            android:layout_marginEnd="15dp"
+            android:layout_marginBottom="15dp" >
+            <TableRow>
+                <ImageView
+                    android:id="@+id/brave_sync_image_not_working"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="center"
+                    android:contentDescription="@null"
+                    android:gravity="center"
+                    android:maxWidth="24dp"
+                    android:paddingRight="5dp"
+                    android:adjustViewBounds="true"
+                    android:scaleType="centerCrop"
+                    android:src="@drawable/ic_warning_circle"
+                    android:tint="@android:color/black" />
+                <TextView
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:gravity="left"
```

Comment:
it should be android:gravity="start" instead of android:gravity="left". It's for native RTL support.